package/rubygem-packaging_rake_tasks.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Fri May 10 10:49:59 UTC 2019 - David Diaz <dgonzalez@suse.com>
+
+- Add support for Suse's Jira tags (bsc#1134704)
```

Comment:
SUSE, please :)

Reply:
I just copied it from the referenced mail in #49 :)

Updated.